modules/tinymce/src/plugins/fullscreen/main/ts/core/Actions.ts

```diff
@@ -79,13 +94,70 @@ const viewportUpdate = WindowVisualViewport.get().fold(
   }
 );
 
-const toggleFullscreen = (editor: Editor, fullscreenState: Cell<any>) => {
+const getFullscreenRoot = (editor: Editor): SugarElement<Element> => {
+  const elem = SugarElement.fromDom(editor.getElement());
+  return SugarShadowDom.getShadowRoot(elem).map(SugarShadowDom.getShadowHost).
+    getOrThunk(() => SugarBody.getBody(Traverse.owner(elem)));
+};
+
+const getFullscreenElement = (root: DocumentOrShadowRoot) => {
+  if (root.fullscreenElement !== undefined) {
+    return root.fullscreenElement;
+  } else if ((root as any).msFullscreenElement !== undefined) {
+    return (root as any).msFullscreenElement;
+  } else if ((root as any).webkitFullscreenElement !== undefined) {
+    return (root as any).webkitFullscreenElement;
+  } else {
+    return null;
+  }
+};
+
+const getFullscreenchangeEventName = () => {
+  if (document.fullscreenElement !== undefined) {
+    return 'fullscreenchange';
+  }
```

Comment:
Why have the else on a new line, as that's not the normal codestyle we use? If there's no reason could you please collapse it to keep things consistent?

Reply:
Sorry, that was because I was commenting/uncommenting things when I was trying to get IE working. I forgot to fix it.